src/Mod/Sketcher/Gui/DrawSketchController.h

```diff
@@ -686,9 +699,10 @@ class DrawSketchController
     }
     //@}
 
+
 private:
-    ColorManager colorManager;
     OnViewParameterVisibilityManager ovpVisibilityManager;
+    ColorManager colorManager;

```

Comment:
Not sure why this change. It was alphabetically ordered... I do not see any initialisation issue...

Reply:
No use, at some point I wanted to make ovpVisibilityManager public so I moved it up, then thought making it public was probably not a good idea ;)